src/objects/zcl_abapgit_object_ectd.clas.abap

```diff
@@ -267,8 +267,8 @@ CLASS zcl_abapgit_object_ectd IMPLEMENTATION.
 
           ls_current_changed = get_change_information( <ls_version_info> ).
 
-          IF    ls_current_changed-ldate > ls_last_changed-ldate
-            AND ls_current_changed-ltime > ls_last_changed-ltime.
+          IF     ls_current_changed-ldate >= ls_last_changed-ldate
+             AND ls_current_changed-ltime >  ls_last_changed-ltime.
```

Comment:
I think this wont work, in case `3 > 2 and 10 > 20` it will return false. Need something like 
`IF ldate > ldate OR ( ldate = ldate and ltime > ltime )`

Reply:
Oh, I see. I'll fix it tomorrow.